_overviews/core/nightlies.md

```diff
@@ -0,0 +1,87 @@
+---
+layout: singlepage-overview
+title: Nightly Versions of Scala
+permalink: /overviews/core/:title.html
+---
+
+We regularly publish nightly versions of both Scala 2 and 3 so that users can preview and test the contents of upcoming releases.
+
+We informally call them "nightly" versions, but technically it's a misnomer. A so-called “nightly” is built for every merged PR.
```

Comment:
should we call them something else then, eg "integration builds"?

Reply:
I think that ship has sailed. The terminology is well established in Scala, and it's used by other programming languages as well. And it's baked into scala-cli, which supports e.g. `-S 2.nightly`. I've never seen it result in confusion of any consequence.

Reply:
This is wrong in Scala 3. We publish a nightly everyday at 3AM UTC and if a new commit was merged to main after the release of the previous nightly release.

Reply:
For Scala 3, they are actually correctly named 😅

Reply:
fixed, good to know!